public/app/plugins/datasource/loki/components/LokiLabel.tsx

```diff
@@ -24,6 +24,7 @@ export const LokiLabel = forwardRef<HTMLElement, Props>(
   ({ name, value, hidden, facets, onClick, className, loading, searchTerm, active, style, ...rest }, ref) => {
     const theme = useTheme();
     const styles = getLabelStyles(theme);
+    const matcher = searchTerm ? new RegExp(searchTerm, 'i') : undefined;
```

Comment:
Nit: Highlighter supports passing strings so there's no need to convert it a RegExp.